takwimu/templates/takwimu/_includes/navbar/support_services.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'takwimu/_includes/navbar/_dropdown.html' %}
+
+{% load staticfiles %}
+
+{% block dropdown_id %}nav-about{% endblock %}
+
+{% block dropdown_title %}Support Services{% endblock %}
+{% block dropdown_desc %}Cum sociis natoque penatibus et magnis dis parturient
+  montes, nascetur ridiculus mus. Vivamus sagittis lacus vel augue laoreet
+  rutrum faucibus dolor auctor.{% endblock %}
+
+{% block dropdown_content %}
+  <div class="row w-100">
+    <div class="col-4">
+      <h6 class="font-weight-bold">Data Services</h6>
+    </div>
+    <div class="col-4">
+      <h6 class="font-weight-bold">Value Added Services</h6>
+      <ul>
+        {% for service in support_services %}
+          <li>For {{ service.title }}</li>
+        {% endfor %}
+      </ul>
+    </div>
+
+    <div class="col-4">
+      <h6 class="font-weight-bold">Help and support</h6>
+      <div class="mt-2 d-inline-block p-2 social_media">
+        <ul class="list-unstyled" style="">
+          <li class="px-2">
+            <a href="{{ settings.takwimu.SocialMediaSettings.twitter }}" target="_blank"><i class="fab fa-twitter fa-fw"></i></a>
+          </li>
+          <li class="px-2">
+            <a href="{{ settings.takwimu.SocialMediaSettings.facebook }}" target="_blank"><i class="fab fa-facebook fa-fw"></i></a>
+          </li>
+          <li class="px-2">
+            <a href="{{ settings.takwimu.SocialMediaSettings.instagram }}" target="_blank"><i class="fab fa-instagram fa-fw"></i></a>
+          </li>
+          <li class="px-2">
+            <a href="{{ settings.takwimu.SocialMediaSettings.github }}" target="_blank"><i class="fab fa-github fa-fw"></i></a>
+          </li>
+        </ul>
+      </div>
+      <br>
+      <ul class="mt-2">
+        <li><a href="/contact-us">Contact Us</a></li>
+        <li><a href="https://takwimu.zendesk.com/" target="_blank">Support</a></li>
+        <li><a href="/about">About Us</a></li>
+      </ul>
+    </div>
+
+
+  </div>
+  <style>
+  .social_media{
+    background-color: #2c73c5;
+  }
+
+  .social_media ul {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .social_media li {
+    display: inline;
+  }
+
+  .social_media li a:hover {
+    color: #ffffff;
+  }
```

Comment:
Does this only happen on here? Could be better selected using id defined [here](https://github.com/TakwimuAfrica/TAKWIMU/pull/181/files#diff-90127d9c15a5571a994266a9286a0fdaR5) that also needs updating.